v1/tester/runner_test.go

```diff
@@ -750,71 +749,49 @@ func TestRunnerWithCustomBuiltin(t *testing.T) {
 	})
 }
 
-func TestRunnerWithBuiltinErrors(t *testing.T) {
-	const ruleTemplate = `package test
-	import rego.v1
-
-	test_json_parsing if {
-      x := json.unmarshal("%s")
-	  x.test == 123
-	}`
-
-	testCases := []struct {
-		desc          string
-		json          string
-		builtinErrors bool
-		wantErr       bool
-	}{
-		{
-			desc:          "Valid JSON with flag enabled does not raise an error",
-			json:          `{\"test\": 123}`,
-			builtinErrors: true,
-		},
-		{
-			desc:          "Invalid JSON with flag enabled raises an error",
-			json:          `test: 123`,
-			builtinErrors: true,
-			wantErr:       true,
-		},
-		{
-			desc: "Invalid JSON with flag disabled does not raise an error",
-			json: `test: 123`,
-		},
```

Comment:
Deleting test-cases like this warrants some explanation, I think. Unless these are conditioning dead code, it's possible that we might now not be detecting a regression.